.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
   - if [[ "$BOARD" =~ "arduino:samd:" ]]; then
       arduino --install-boards arduino:samd;
       arduino --install-library WiFi101;
```

Comment:
Got it. We plan to merge in this PR, but we will remove the #ifdef around the Serial.setDebugOutput(true). If this is blocking you in any way, please let us know.